### ISSUE TITLE
LeadingZero raised unexpected error

### ISSUE THREAD

**Issue by Boldewyn** [brigade/scss-lint#194]
I've just installed scss-lint and ran with the very first try into this problem:

```
$ scss-lint src/sass/
SCSSLint::Linter::LeadingZero raised unexpected error linting file src/sass/frontend/_button_styles.scss: 'undefined method `[]' for nil:NilClass'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter/leading_zero.rb:17:in `visit_script_number'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:36:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:123:in `visit_children'
/var/lib/gems/1.9.1/gems/sass-3.3.4/lib/sass/tree/visitors/base.rb:38:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:115:in `visit'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/linter.rb:18:in `run'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/runner.rb:43:in `block in find_lints'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/runner.rb:38:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/runner.rb:38:in `find_lints'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/runner.rb:22:in `block in run'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/runner.rb:21:in `each'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/runner.rb:21:in `run'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/lib/scss_lint/cli.rb:98:in `run'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/bin/scss-lint:8:in `block in <top (required)>'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/bin/scss-lint:6:in `tap'
/var/lib/gems/1.9.1/gems/scss-lint-0.25.1/bin/scss-lint:6:in `<top (required)>'
/usr/local/bin/scss-lint:23:in `load'
/usr/local/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/causes/scss-lint/issues

$ ruby --version
ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]

$ sass --version
Sass 3.3.4 (Maptastic Maple)

$ scss-lint --version
scss-lint 0.25.1

$ uname -a
Linux localhost 3.11-2-amd64 #1 SMP Debian 3.11.8-1 (2013-11-13) x86_64 GNU/Linux
```

Any ideas, what that could cause? The mentioned SCSS file contains only some variable definitions (with `!default`) and placeholder selectors.
